packages/edit-navigation/src/components/block-placeholder/index.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * WordPress dependencies
+ */
+/**
+ * WordPress dependencies
+ */
```

Comment:
Sorry missed this one yesterday.

Reply:
Haha! They just keep coming. I spotted a few others in the codebase too 😅 

I also missed the two comments you made here:

> - Should we move hooks from use-navigation-entities.js into the hooks directory?
> - I think we already have few utility methods from menu-items-to-blocks.js in store/utils.js so we can re-use them.

I think the `useNavigationEntities` isn't quite general purpose enough to be moved.

Second point is a good observation that I missed, I'll take a look at that now.